tests/assets/apps/configurable_hello_world.py

```diff
@@ -1,11 +1,46 @@
+from typing import Type
+
 from steamship.app import Invocable, InvocableResponse, create_handler, post
+from steamship.app.config import Config
 
 
 class HelloWorld(Invocable):
+    # Package configuration is always defined by a Pydantic object which derives from
+    # the base `Config` object.
+    class HelloWorldConfig(Config):
+        greeting: str
+
+        # The casing of config variables is preserved exactly as the Steamship package author defines them.
+        # Snake case is permitted
+        snake_case_config: str
+        # Camel case is permitted
+        camelCaseConfig: str  # noqa: N815
```

Comment:
I was wondering if the engine would accept snake_case without camelCase transformation. It is. Thanks for checking!